nodes/textures/iorpreset.py

```diff
@@ -0,0 +1,71 @@
+"""A BlendLuxCore node to provide index of refraction preset values for
+    LuxCoreRender in Blender"""
+# <pep8 compliant>
+import bpy
+from bpy.props import FloatProperty, StringProperty
+from .. import LuxCoreNodeTexture
+from ...operators import ior_presets
+
+
+class LuxCoreNodeIORPreset(LuxCoreNodeTexture):
+    """ Index of Refraction Preset node """
+    bl_label = "IOR Preset"
+    bl_width_min = 180
+    bl_width_default = 180
```

Comment:
Having both bl_width_min and bl_width_default at the same value is a bit redundant.
By the way, I think we should use bl_width_default everywhere because it means more freedom for the user to customize the node tree.
Currently most nodes are using bl_width_min, so I will eventually have to change them all.

Long story short: Remove bl_width_min.